homeassistant/components/permobil/sensor.py

```diff
@@ -209,6 +241,15 @@ def __init__(
             f"{coordinator.p_api.email}_{self.entity_description.key}"
         )
 
+    @property
+    def native_unit_of_measurement(self) -> str | UnitOfLength | None:
+        """Return the unit of measurement of the sensor."""
+        if self.entity_description.key in DYNAMIC_UOM_SENSORS:
```

Comment:
How much more sensors will you add with dynamic uom? If this is the only one in the foreseeable future, I'd recommend going for the smaller implementation I put in that other PR to keep the code more readable